### PR TITLE
fix(node): read consistency level fix

### LINKED ISSUE #1565: bug(node): `readConsistencyInterval` not passed down in `vectordb`
### LanceDB version

0.4.20

### What happened?

`readConsistencyInterval` doesn't seem to be properly passed down. This is likely never 
receiving the correct argument here:

https://github.com/lancedb/lancedb/blob/9c259981108768e6ab01cb5f7948389266f2b197/rust/ffi/node/src/lib.rs#L51-L52

We should test this like we do in the newer SDK:

https://github.com/lancedb/lancedb/blob/9c259981108768e6ab01cb5f7948389266f2b197/nodejs/__test__/table.test.ts#L540

### Are there known steps to reproduce?

_No response_

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
PR fixes #1565

### CODE REVIEW

**Comment by gagan-bhullar-tech** [comment]
@wjones127 please review.

File: node/src/test/test.ts [hunk 1]
```diff
@@ -93,6 +93,29 @@ describe("LanceDB client", function () {
...
+      const db1 = await lancedb.connect({ uri });
+      const table1 = await db1.openTable("vectors");
+
+      const db2 = await lancedb.connect({
+        uri,
+        readConsistencyInterval: 0
+      })
+      const table2 = await db2.openTable("vectors");
+
+      await table1.add([
```
Comment (wjones127) [inline node/src/test/test.ts:108]: We should make this test more robust by asserting the state before the change as well:
```suggestion
      assert.equal(await table2.countRows(), 2);
      await table1.add([
```

**Review by wjones127 — CHANGES_REQUESTED** [review]

**Comment by gagan-bhullar-tech** [comment]
@wjones127 Updated PR. Please review.

**Review by wjones127 — APPROVED** [review]

**Comment by wjones127** [comment]
Thanks @gagan-bhullar-tech !

**Pull request merged**